### PR TITLE
my first html file

### PR DESCRIPTION
**Pull request by wajeeh-syed**

### CODE REVIEW

File: module1-the-basics/index.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<html lang="en">
```
Comment (hafeez-syed) [inline module1-the-basics/index.html:2]: missing `<!doctype>`. @wajeeh-syed please action.

File: module1-the-basics/index.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<html lang="en"> 
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    Wajeeh's first HTML file
+</body>
+</html>
```
Comment (hafeez-syed) [inline module1-the-basics/index.html]: line ending missing

**Review by hafeez-syed — CHANGES_REQUESTED** [review]

File: module1-the-basics/index.html (hunk 1 above)
Comment (wajeeh-syed) [reply module1-the-basics/index.html:2]: fixed

File: module1-the-basics/index.html (hunk 2 above)
Comment (wajeeh-syed) [reply module1-the-basics/index.html]: line added

**Review by hafeez-syed — APPROVED** [review]
Good job!

**Pull request merged**